### DISCUSSION TITLE
기획 레퍼런스 창구

### DISCUSSION THREAD

**Discussion by CreatorLDH** [GG-Studio-990001/Grayed-Game#56] [Ideas]
방금 회의에서 나온 이야기대로,
레퍼런스 창구를 깃헙에도 개설했습니다.

따로 ppt에 기입하시지 않더라도 여기에 남겨주시면 제가 옮기거나 활용하겠습니다.

열심히 하겠습니다 감사합니다:D

https://docs.google.com/presentation/d/1Rsym-yI_SNIOXCTHe5pZ2GWpPMmHCYJAUAcb2fcMF-0/edit?usp=drive_link

**Comment by wooyn730** [comment]
### 아무 생각 조금
- TRPG 캐릭터
   - 요즘 TRPG에 관심이 있는데 (마침 이전 기획에 언급되기도 했고) TRPG는 주사위 숫자에 따라 판정을 하니까 주사위를 굴려서 상호작용하는 TRPG NPC가 있으면 재밌을 것 같습니다
   - ex) 협상을 해야할 때 1\~2: 협상 실패, 3\~4: 어려운 성공, 5~6: 쉬운 성공 요런...
- 맥거핀
   - 중요해보이는 데 아무것도 아닌 거
   - 비싼 황금 열쇠를 파는데 사실 아무 데도 쓸 데가 없다던가..
- 퀴즈할 때 써먹을만한 요소
  - 모스부호, 애너그램, 점자, 진법, 무전기 교신?용어
  - 최현배 선생님의 한글 풀어쓰기 (이거 뭔가 조금은 알아볼 수 있겠으면서도 외계어 느낌 나서 재밌어요 / 근데 현지화 생각하면 못쓸듯 ㅎㅎ)
![image](https://github.com/GG-Studio-990001/GameOver/assets/80298207/8c17bb27-a6e3-4b32-b998-821d91f816c4)
- 피아노
  - 밟아서 연주할 수 있는 피아노
  - 오버워치도 대기실에 피아노 있는데 유저들이 치면서 [재밌게(?) 놀아서](https://www.youtube.com/watch?v=Wcl1zHsa9es) 이런 .. 쓸모 없지만 귀여운 요소 넣으면 재밌겠다 싶은 생각 ㅎㅎ

**Comment by fkdl0048** [comment]
## @fkdl0048 

- 기획안의 2D 탑뷰 이후에 마우스를 획득한다면?
  - 마을의 나무나 바닥을 클릭하면 먼지, 나무 흔들림 이펙트 재생
  - 메인 시스템과 마을의 결합을 생각
  - 하스스톤 게임장의 클릭과 같은 느낌

**Comment by Songkihwa** [comment]
**간단한 레퍼 의견(이라고 쓰고 저두 아무 생각입니다)** 

- 파이어 보이와 워터 걸 : 서브시스템으로 넣어도 크게 부담이 없지 않을까 해서 의견 드립니다! 사실 제가 좋아했던 게임이였어서요. 퀘스트 중에 길이나 문을 찾아야한다면 이 게임으로 활용해봐도 좋을 것 같아요.

- PPT 내용 중 GTA 관련 : 이 차는 제껍니다 할 때 차 창문 팔꿈치로 깨는 것도 꼭 들어가면 재밌을 것 같습니다ㅋㅋㅋ 제가 즐겨 했었어서요 ...

- PPT 내용 중 타이쿤 관련 : 내용과는 조금 다르긴 하지만 슈의 라면가게 .. 아시나요? 이것도 비슷하게 들어가면 재밌을 것 같습니다! 그 게임 속 대사라던지 비슷하게 활용하면 누구나 한번쯤 해봤을 어렸을 적 했던 추억 게임이라 다들 좋아하지 않을까 해서 이렇게 의견드려봅니다 ㅎㅎ 

-정안님 의견 관련 : 나무에 올라탈 수 있는 것도 좋을 것 같아요. 나무를 올라타서 나무 안에 숨은 요소들을 찾는다거나, 아니면 나무를 흔들어서 숨겨진 뭔가를 떨어뜨린다거나 하는 것들이요!

**Comment by CreatorLDH** [comment]
**// issue에서 discussion 이동 //**

**Comment by fkdl0048** [comment]
- 서브시스템 게임의 역발상 관련
  - 많이 나오는 옛날 게임의 특징을 역 발상하여 다영한 아이디어 도출
  - 용사, 탑, 마녀, 드래곤, 공주의 아이템을 사용하여 반대인 공주를 납치?, 독 포션을 만드는 마녀? 등 역 발상으로 생각해도 좋을 듯

**Comment by fkdl0048** [comment]
- 현재 NPC 수달처럼 게임 개성이 강한 캐릭터처럼 이후 등장하는 챕터에서 사운드에 진심인 캐릭터?
  - 사운드팀원이 있으니 사운드 NPC? 이후 다른 챕터에서 리듬게임을 활용
  - 사운드팩을 가져다 달라는 퀘스트 등
  - UI와 연계도 가능

**Comment by wooyn730** [comment]
기획회의 들으면서 생각한 세이브 관련!

- 세이브 포인트
  - 세이브 포인트 관련해서 생각해봤는데 기억과 관련한 상징이 있으면 좋겠다고 생각했습니다. 그래서 생각한 것 중 하나가 컴퓨터 RAM(Random access memory)와 숫양(ram)이 철자가 같으니 세이브 포인트에 양이나 양털 같은 거 두기 ㅎㅎ...
  - 아니면 호두(먹으면 기억력 좋아진다고 해서)나 사진기(사진에는 순간을 담을 수 있으니까), 싸인펜/공책(화이트데이 저장 시스템) 같은 생각도 해봤습니다.
  - 문구는 "초록색 털을 가진 양이 여기 있다는 것을 기억했다.", "초록색 털을 가진 양: 내가 너가 여기 있었다는 것을 기억할게" 이런 기억이나 잊지 않겠다 같은 키워드 넣기...

- 세이브 관련 이스터에그?
  - 언더테일 초반에 토리엘이 전화로 시나몬과 버터스카치 중에 뭘 좋아하냐고 묻고 선택지를 고르면, 이후에 세이브 안하고 다시 켜서 해당 선택지를 다시 마주하면 너는 시나몬(아까 고른 선택지)을 좋아할 것 같다라는 문구가 추가됩니다. 신기하고 기분 좋았던 경험이었습니다.
  - 세이브를 안해서, 또는 게임 오버로 인해 특정 상황을 다시 반복해야할 때 플레이어는 지루함을 느낄 수 있는데 이런 소소한 포인트를 넣어두면 재미있겠습니다~

**Reply by fkdl0048** [reply]
세이브 포인트 아이디어는 게임이라는 소재보다 컴퓨터에 관련한 소재를 말씀해주셨는데(or 저장에 관한 매개체)

- 컴퓨터(게임의 본질?)에 관한 소재는 설명 불가능한, 초월적인? 게임의 잊혀짐에 관한 스토리에서 벗어날 수 있겠다는 생각이 듭니다.
  - 그래서 이후로도 컴퓨터쪽 관련은 사용하면 현재 게임 스토리에서 어색한, 동 떨어져 있는 스토리가 될 수 있을 것 같습니다
  - 따라서 회의 때 나온 잊혀짐이나 자동 세이브를 어떻게 처리하느냐가 관건일 것 같습니다?
- 지금 생각으론 특별한 일이 아니라면 세이브 파일 하나로 진행되는게 좋겠다는 생각입니다.(자동저장으로), 게임 특성상 사용자가 죽어서 되 돌아가는? 일이 없기 때문에 선형적인 스토리, 게임의 특별함을 보여주기에 자동 저장으로만 처리하는게 좋겠다는 생각..
- 세이브 관련 이스터에그도 회의 때 나온 것처럼 세이브나 로드 자체를 기믹으로 활용하는게 더 재밌을 것 같은? 느낌입니다
- 하나의 챕터에서 활용되기 좋은 B레벨의 펴즐정도, 마찬가지로 자주 사용하는 기믹이 되면 방탈출 게임이 되어버리는..

**Comment by fkdl0048** [comment]
[재밌어보이는 핀볼 연출](https://www.reddit.com/r/godot/comments/17hiaje/just_a_quick_physics_engine_test_everyone_says/)
